ci/ci-deploy-infra.sh

```diff
@@ -38,21 +38,25 @@ function install_kvm_packages {
   scp_base="sshpass -p ${hvpass} scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=quiet "
 
   # scp packages to hypervisor, remove existing, then install new ones
-  ${ssh_base} ${hvuser}@${hvip} rm -f cloudstack-\*
```

Comment:
As discussed, apparently that's not true :-) We'll clean up the old cloudstack scripts later.